downstream/modules/dev-guide/con-content-workflows.adoc

```diff
@@ -0,0 +1,20 @@
+
+[id="con-content-workflows_introduction"]
+
+
+= About content workflows
+
+
+[role="_abstract"]
+Before {PlatformName} 2.0, an automation content developer may have needed so many Python virtual environments that they required their own automation in order to manage them. To reduce this level of complexity, {PlatformNameShort} 2.0 is moving away from virtual environments and using containers, referred to as {ExecEnvName}, instead, as they are straightforward to build and manage and are more shareable across teams and orgs.
+
+As {ControllerName} shifts to using {ExecEnvName}, tools like {Navigator} and {Builder} ensure that you can take advantage of those {ExecEnvName} locally within your own development system.
```

Comment:
Should we refer to the product name as Execution environment builder (https://docs.google.com/spreadsheets/d/1l8YLUY-AFHYgxQKNe2NvRtk85-MenYcr-BaMAYFZjfs/edit#gid=0) and teh command line tool as `ansible-builder` ?